lua/lspconfig/server_configurations/vdmj.lua

```diff
@@ -1,7 +1,7 @@
 local util = require 'lspconfig.util'
 
 if vim.fn.has 'nvim-0.5.1' == 0 then
```

Comment:
That check can be removed, as lspconfig itself should require 0.5.1 (and 0.6.0 next week).

Reply:
We should also concurrently make that clear in the readme, maybe not this PR

Reply:
That was planned anyway, right?

(Although I'd probably synchronize with the release: "if on Neovim master, _stay_ on lspconfig master; if on 0.6.0, stay on tag 0.1.0; if on 0.5.1 or earlier, you're on your own -- no support")

Reply:
Yes, I plan to tag on every major release of neovim

Reply:
Getting OT now but on the topic of tagging lspconfig releases - have you considered how to deal with situations where a language server introduces a change that would necessitate a change in lspconfig (like, package name change or new args, etc.)? Pinning the lspconfig version together with neovim releases might make this difficult to handle. Just a thought!

(I can only remember one case where a server cmd have had a breaking change, [solang](https://github.com/neovim/nvim-lspconfig/pull/1388))

Reply:
I wonder if it'd make sense to find a way to have the server configs always up to date, and at the same time encourage/allow pinning the lspconfig plugin version. Maybe this problem will disappear once things are added to core (I understand this is the plan longterm)?

Reply:
In practice, it won't matter. Tags will only be guaranteed for the lifetime of the release they correspond to. I won't backport server changes to say the tag corresponding to the neovim 0.6 release when 0.7 comes out.

Reply:
Ah gotcha! So it will be similar to like release branches, which will be subject to updates for as long as the corresponding neovim release is in "LTS" if you will. Makes sense :+1:

Reply:
Exactly, it would be madness otherwise. Plus people are free to override setup in their configs if they really want to freeze things.

Reply:
>  it would be madness otherwise.

<img width="150" alt="Screenshot 2021-11-27 at 15 51 20" src="https://user-images.githubusercontent.com/6705160/143686301-2579a322-1f38-4e77-9991-e49ef991c2d5.png">